src/config/logger.ts

```diff
@@ -2,8 +2,12 @@ import Logger, {levelFromName} from 'bunyan';
 import bformat from 'bunyan-format';
 import filteringStream from '../util/filteringStream'
 
+// For any Micros env we want the logs to be in JSON format.
+// Otherwise, if local development, we want human readable logs.
+const outputMode = process.env.MICROS_ENV ? 'json' : 'short';
+
 // add levelInString to include DEBUG | ERROR | INFO | WARN
-const formatOut = filteringStream(bformat({outputMode: 'bunyan', levelInString: true}));
+const formatOut = filteringStream(bformat({ outputMode, levelInString: true }));
```

Comment:
I wonder what's the difference between 'json'  and 'bunyan' here? Did you test it in micros?

Reply:
Looking at the code for bunyan-format, they're the same except for `json` which has indentation (multiple line) vs bunyan which is all one line.  Literally `JSON.stringify(log)` vs `JSON.stringify(log, null, 2)`.